pkgs/top-level/all-packages.nix

```
@@ -17300,6 +17300,8 @@ with pkgs;
 
   global = callPackage ../development/tools/misc/global { };
 
+  glualint = haskell.lib.compose.justStaticExecutables haskellPackages.glualint;
```

Seems like uuagc-cabal doesn't support Cabal >= 3.8. We can work around this by using GHC 9.0 which ships Cabal 3.6.

To allow this you'd need to add an entry to `configuration-ghc-9.0.x.nix like that to disable the broken flag that doesn't ring true for that GHC version:

```nix
  uuagc-cabal = doDistribute (unmarkBroken super.uuagc-cabal);
```

and change this line to

```suggestion
  glualint = haskell.lib.compose.justStaticExecutables haskell.packages.ghc90.glualint;
```

---

Thanks, that worked. I've made a new PR #233112 that incorporates the fix and uses the newest upstream version.